packages/server/src/sdk/app/rows/search/filters.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  FieldType,
+  RelationshipFieldMetadata,
+  SearchFilters,
+  Table,
+} from "@budibase/types"
+
+export function getRelationshipColumns(table: Table): {
+  name: string
+  definition: RelationshipFieldMetadata
+}[] {
+  return Object.entries(table.schema)
+    .filter(entry => entry[1].type === FieldType.LINK)
+    .map(entry => ({
+      name: entry[0],
+      definition: entry[1] as RelationshipFieldMetadata,
+    }))
+}
+
+export function getTableIDList(
+  tables: Table[]
+): { name: string; id: string }[] {
+  return tables
+    .filter(table => table.originalName)
```

Comment:
For completeness, do you also want to filter on the _id field here? Because you use it in the map below.